test/units/staking/stakeManager/StakeManager.test.js

```diff
@@ -1192,6 +1193,126 @@ contract('StakeManager', async function (accounts) {
     })
   })
 
+  describe('checkSignature() with votes', function () {
```

Comment:
just "checkSignature"

Reply:
Ah, no, there is already "checkSignatures" describe block. You must move entire test case inside that scope. The structure i would make would be:
```
describe('with votes')
     describe('with 4 validators')
          describe('when 2/3 + 1 vote yes`)
               it('checkpoints')
          describe('when 2/3 vote yes')
               it('reverts')
```

Reply:
okay, i'll change it.

Reply:
done